web/modules/custom/collection/src/Entity/NodeCollectionContentTrait.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\collection\Entity;
+
+/**
+ * Reusable methods for node collection content.
+ */
+trait NodeCollectionContentTrait {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCollection(): CollectionInterface {
```

Comment:
Is this 079f1ad036c8fa2d083b595dac4a173a971a15ea what you mean? I'm not sure I've understood.

Reply:
No I mean about splitting the `CommunityContentBase.php` into two different traits instead of a single one. One of them cannot be used without the other one. It is the `NodeCollectionContentTrait` and the `NodeGroupContentTrait`. However, if one would include the current one without including the `NodeGroupContentTrait`, the `::getCollection` method would fail because it calls for `::getGroup` which is not in this trait anymore. 
On the contrary, the `::getGroup` and `::getCollection` both target all content that are part of an OG group since they either belong to a collection or a solution and even solutions belong to a collection.

Reply:
But in 079f1ad, I'm using only one trait. Don't get your point.